references/gcp-sa-auth-shared-flow/sharedflowbundle/resources/jsc/CheckConfigAndExtractSAKey.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var scope = context.getVariable('gcp.scopes');
+var targetAudience = context.getVariable('gcp.target_audience');
+var serviceAccountKeyJson = context.getVariable('private.gcp.service_account.key');
+
+// Check if required variables are present
+if (!targetAudience && !scope) {
```

Comment:
this is a falsy check - the check will fail not only if targetAudience or scope are undefined, but also if they are "", null, 0 or false. Is it possible somebody would want empty string for audience or scopes (e.g. to only get default scopes)?